ast/misc.go

```diff
@@ -690,6 +691,29 @@ func (n *AnalyzeTableStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+// AnalyzeIndexStmt is used to create index statistics.
+type AnalyzeIndexStmt struct {
```

Comment:
I think we don't need a special ast to hanlde analyze indices. We can reuse the analyze table stmt and add a type to specify the analyze class.